tests/integration_tests/test_run.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python3
+"""
+"""
+
+import json
+import logging
+import os
+import shutil
+from pathlib import Path
+from unittest import TestCase
+
+import flywheel_gear_toolkit
+import pytest
+from flywheel_gear_toolkit.utils.zip_tools import unzip_archive
+
+import run
+
+log = logging.getLogger(__name__)
+
+fs_dir = Path("/usr/local/freesurfer/")
+subjects_dir = Path(fs_dir / "subjects")
+
+
+def install_gear(zip_name):
+    """unarchive initial gear to simulate running inside a real gear.
+
+    This will delete and then install: config.json input/ output/ work/
+
+    Args:
+        zip_name (str): name of zip file that holds simulated gear.
+    """
+
+    gear_tests = "/src/tests/data/gear_tests/"
+    gear = "/flywheel/v0/"
+    os.chdir(gear)  # Make sure we're in the right place (gear works in "work/" dir)
+
+    print("\nRemoving previous gear...")
+
+    if Path(gear + "config.json").exists():
+        Path(gear + "config.json").unlink()
+
+    for dir_name in ["input", "output", "work", "freesurfer"]:
+        path = Path(gear + dir_name)
+        if path.exists():
+            shutil.rmtree(path)
+
+    print(f'\ninstalling new gear, "{zip_name}"...')
+    unzip_archive(gear_tests + zip_name, gear)
+
+    # move freesurfer license and subject directories to proper place
+    gear_freesurfer = Path("freesurfer")
+    if gear_freesurfer.exists():
+        gear_license = Path(gear_freesurfer / "license.txt")
+        if gear_license.exists():
+            if Path(fs_dir / "license.txt").exists():
```

Comment:
who's license file?
Is it a "Flywheel" Special? Or AW?

Reply:
Hmmm.  I didn't think about that.  It's my own "flywheel" license.  Does it matter?  Its not like they phone home to say I'm running Freesurfer and anyone can get a license just by asking.

Reply:
True that.

It only matters when debugging with the devs.